draft-ietf-ppm-dap.md

```diff
@@ -1409,18 +1420,25 @@ This message consists of:
 * `prepare_inits`: the sequence of `PrepareInit` messages constructed in the
   previous step.
 
-Finally, the Leader sends a PUT request to
-`{helper}/tasks/{task-id}/aggregation_jobs/{aggregation-job-id}`. The payload
-is set to `AggregationJobInitReq` and the media type is set to
-"application/dap-aggregation-job-init-req".
+Finally, the Leader sends an HTTP PUT request to
+`{helper}/tasks/{task-id}/aggregation_jobs/{aggregation-job-id}` with a media
+type of "application/dap-aggregation-job-init-req" and a body containing the
+`AggregationJobInitReq`.
 
 The Leader MUST authenticate its requests to the Helper using a scheme that
 meets the requirements in {{request-authentication}}.
 
-The Helper's response will be an `AggregationJobResp` message (see
-{{aggregation-helper-init}}. The response's `prepare_resps` must include exactly
-the same report IDs in the same order as the Leader's `AggregationJobInitReq`.
-Otherwise, the Leader MUST abort the aggregation job.
+The Helper responds with HTTP status 201 Created with a body containing an
+`AggregationJobResp` (see {{aggregation-helper-init}}). If the `status` field
+is `finished`, the Leader proceeds onward. Otherwise, if the `status` field is
+`processing`, the Leader polls the aggregation job by sending GET requests to
+the URI indicated in the Location header field, until the `status` is
```

Comment:
Why doesn't this work just like [polling the collection job URI](https://ietf-wg-ppm.github.io/draft-ietf-ppm-dap/draft-ietf-ppm-dap.html#section-4.8.1-15)?

Reply:
No need to change it, but if you think it's worth adjusting collection to match, then let's file an issue to track.

Reply:
Collection job polling actually runs awry of @martinthomson's feedback about 202 being the wrong code, needing Location header, etc. https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/pull/564#discussion_r1701106596.

So I think we should adjust collection to match.